src/app/main/component/eco-news/components/create-edit-news/create-edit-news.component.ts

```diff
@@ -34,7 +34,7 @@ export class CreateEditNewsComponent extends FormBaseComponent implements OnInit
   public month: number = new Date().getMonth();
   public author: string = localStorage.getItem('name');
   public attributes: ActionInterface;
-  public filters: Array<FilterModel>;
+  public filters: FilterModel[] | [] = [];
```

Comment:
Do we need second type of regular array? Empty array value still acceptable as FilterModel[] type

Reply:
done